internal/storage/bucket_handle_test.go

```diff
@@ -467,3 +470,372 @@ func (t *BucketHandleTest) TestUpdateObjectMethodWithMissingObject() {
 
 	AssertTrue(errors.As(err, &notfound))
 }
+
+func (t *BucketHandleTest) TestComposeObjectMethodWithDstObjectExist() {
+	// Checking that destination object exist
+	_, err := t.bucketHandle.StatObject(context.Background(),
+		&gcs.StatObjectRequest{
+			Name: TestObjectName,
+		})
+
+	AssertEq(nil, err)
+
+	// Reading content before composing it
+	rc, err := t.bucketHandle.NewReader(context.Background(),
+		&gcs.ReadObjectRequest{
+			Name: TestObjectName,
+			Range: &gcs.ByteRange{
+				Start: uint64(0),
+				Limit: uint64(len(ContentInTestObject)),
+			},
+		})
+
+	AssertEq(nil, err)
+
```

Comment:
please remove all new lines, its becoming difficult to read. Just keep new lines between test initializaiton, method invocation, assert statetments.

If test initialiation has asserts in it, no need of new lines.  same for other sections.

Reply:
fixed